Description: Fix FTBFS with doxygen 1.15.0 due to test failures.
Author: Bas Couwenberg <sebastic@debian.org>
Forwarded: https://github.com/libgeos/geos/pull/1341
Applied-Upstream: https://github.com/libgeos/geos/commit/e8c6acca2cd023ea699eaa9237695f860e2ee72a

--- a/include/geos/linearref/LengthIndexedLine.h
+++ b/include/geos/linearref/LengthIndexedLine.h
@@ -138,7 +138,7 @@ public:
      * slightly off the line and is equidistant from two different
      * points on the line.
      *
-     * The supplied point does not `*necessarily* have to lie precisely
+     * The supplied point does not *necessarily* have to lie precisely
      * on the line, but if it is far from the line the accuracy and
      * performance of this function is not guaranteed.
      * Use {@link #project} to compute a guaranteed result for points
